test/system/030-run.bats

```diff
@@ -974,4 +974,9 @@ EOF
     run_podman 125 create --name "$randomname/" $IMAGE
 }
 
+@test "podman run --net=host --cgroupns=host with read only cgroupfs" {
+    # verify that the last /sys/fs/cgroup mount is read-only
+    run_podman run --net=host --cgroupns=host --rm $IMAGE sh -c "grep ' / /sys/fs/cgroup ' /proc/self/mountinfo | tail -n 1 | grep '/sys/fs/cgroup ro'"
+}
```

Comment:
Can't do that. You can't `run_podman | anything`. I encourage you to use the test code I already wrote. And I *especially* encourage you to use `hack/bats`:
```console
$ hack/bats 030:"read only cgr"
```
...and to use it against a podman that does not include your fixes. If the test passes against `/usr/bin/podman` or against podman @ main, your test is broken.

Reply:
the pipes are part of the command that runs inside the container.

The test fails when I use an older podman version

Reply:
Oops - sorry, I misread it. I still wish the test was better commented, but I won't block over that.